src/canvas/Canvas.ts

```diff
@@ -1533,8 +1530,7 @@ export class Canvas extends SelectableCanvas implements CanvasOptions {
       point2 = point1.add(new Point(deltaX, deltaY)),
       tl = point1.min(point2),
       br = point1.max(point2),
-      size = br.subtract(tl),
-      isClick = point1.eq(point2);
```

Comment:
this isClick here is not like the ones we removed from events. i prefer to leave it in.

Reply:
Did you read my comment about this?

Reply:
The method is called only when the browser determines it is not a click
So there is no need and we can test that this line is always false to be sure

Reply:
Yes i read it.
There are 2 points.
One is that this pr is open to clear something that was left over from the isClick in the mouse event object, this isn't. This is a different part of code, is not a leftover from the previous PR.

The other is that you can mouse move  as much as you want, but then if you go back an leave the mouse in the same point, this will still return true. You can also trigger a mouse move and end up with the mouse in the same exact position, because pixels are """large""" compared to the sensibility of mouse move, since there are no decimal pixels in mouse events.

For example, when i try to click with the trackpad and my finger moves a little bit, here is what i get

<img width="520" alt="image" src="https://github.com/fabricjs/fabric.js/assets/1194048/bd8f13c3-f4c6-4d52-8f71-21c68e593291">

i added this in the sandbox:

canvas.on('mouse:move', ({ e }) => {
  console.log(e.clientX, e.clientY);
});

and as you see i fire a bunch of mouse move that are always on the same point

Reply:
restored though I do not understand the use case - the test might be incorrect

Reply:
There is no use case as feature.
Is there to prevent misclicks or mis-selections.

Reply:
So i do not understand why the click condition does not cover it
For a chat